packages/kubernetes/template/script.sh

```diff
@@ -2,25 +2,31 @@
 
 set -euo pipefail
 
-# Populate VERSIONS array with 1.20+ and 1.19 and 1.18 latest versions available
+# Populate VERSIONS array latest kURL-support versions (1.19, 1.20, 1.21) available
 VERSIONS=()
 function find_available_versions() {
     docker build -t k8s - < Dockerfile
-    VERSIONS=($(docker run k8s apt list -a kubelet 2>/dev/null | grep -Eo '1\.[2-9][0-9]\.[0-9]+' | sort -rV | uniq))
-    echo "Found ${#VERSIONS[*]} versions for Kubernetes 1.20+: ${VERSIONS[*]}"
+    # VERSIONS=($(docker run k8s apt list -a kubelet 2>/dev/null | grep -Eo '1\.[2][0-9]\.[0-9]+' | sort -rV | uniq))
```

Comment:
this restricts to only 1.2x  We can keep it as is and pull in the needed versions only.